lib/deserializers.php

```diff
@@ -161,3 +161,31 @@ function elementList(Reader $reader, $namespace = null) {
     return $values;
 
 }
+
+/**
+ * @param Reader $reader
+ * @param $popo
+ * @param $namespace
+ * @return null|$popo
+ */
+function popoObject(Reader $reader, $popo, $namespace) {
+    if ($reader->isEmptyElement) {
+        $reader->next();
+        return null;
+    }
+
+    $reader->read();
+    do {
+
+        if ($reader->nodeType === Reader::ELEMENT && $reader->namespaceURI == $namespace) {
```

Comment:
Future idea: add the ability to parse annotations and let people create classes like this:

```php

class MyVO {

   /**
    * @map {http://namespace}elem
    */
   public $foo;

}
```